### ISSUE TITLE
Unable to run vagrant.exe from WSL: "stdin is not a tty"

### ISSUE THREAD

**Issue by poma** [mitchellh/vagrant#8900]
### Vagrant version
1.9.7 in both windows and WSL. Version 1.9.6 does not contain this bug, so it was introduced in one of the recent commits.

### Host operating system
Windows 10 and WSL Ubuntu 16.04

### Expected behavior
Vagrant displays version

### Actual behavior
Error: "stdin is not a tty"

### Steps to reproduce
In WSL:
```
/mnt/c$ which vagrant.exe
/mnt/c/vagrant/bin/vagrant.exe

/mnt/c$ vagrant.exe version
stdin is not a tty

/mnt/c$
```

In cmd everything works fine:
```
C:\>vagrant.exe version
Installed Version: 1.9.7
Latest Version: 1.9.7

You're running an up-to-date version of Vagrant!
C:\>
```

### References
This issue is the root cause for #8858

**Comment by poma** [comment]
I've found that issue is caused by transition to winpty/cygwin. I've tried to install v1.9.7 and then replace all files to older ones from v1.9.6. So the only difference between working and broken installation is:

- bin/vagrant.exe
- embedded/bin/

**Comment by chrisroberts** [comment]
This has been resolved in the 1.9.8 release. If you are still experiencing this issue after upgrade, please open a new issue. Thanks!
